01_GettingStartedWithDataLab/01_JupyterPythonSQL101.ipynb

```diff
@@ -1075,15 +1180,21 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 40,
+   "execution_count": 6,
    "metadata": {},
    "outputs": [
     {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "CPU times: user 61 ms, sys: 5.2 ms, total: 66.2 ms\n",
-      "Wall time: 19.1 s\n"
+     "ename": "queryClientError",
```

Comment:
The NB as pushed to your branch has a hanging state where the query to be executed is trying to go to lsst_sm.simdr1. Please select "Restart kernel and run all cells", then save again, and use this ipynb file for a new git pull.